R/checks.R

```diff
@@ -537,6 +517,7 @@ val_class_or_null <- function(x, cls = "numeric", where = NULL) {
   }
   invisible(NULL)
 }
+# TODO remove this once finetune is updated
```

Comment:
Note to myself and @simonpcouch to make similar changes to finetune so that we can remove these three checking functions